src/lib.rs

```diff
@@ -50,13 +59,13 @@ pub async fn sync_ldap_users_to_zitadel(config: Config) -> Result<()> {
 
 	sync_handle.await??;
 
-	zitadel.update_users(changed).await?;
-
-	if !deactivate_only {
+	if !deactivate_from_ldap {
```

Comment:
This will not be replacing `deactivate_only`, it's a new source. You can have `deactivate_only` mode *and* this source at the same time.

Reply:
There is no change here, the code just switched lines, see five lines bellow.
(Shouldn't have mixed this here, as it's not related to my task, just when I was reading the code, I saw this logic and thought that the update should happen after importing new users, not before, just in case we are updating newly created ones.)